applications/views.py

```diff
@@ -30,11 +30,10 @@ def page(request, pk, page_num):
     page = Wizard(application, form_specs).get_page(page_num)
 
     if request.method == "GET":
-        ctx = {
-            "application": application,
-            "form": page.get_unbound_form(),
-            "page": page,
-        }
+        form = page.get_unbound_form()
+        ctx = page.form_spec.template_context(form)
+        ctx["application"] = application
+        ctx["page"] = page
```

Comment:
Now that we can splat dictionaries, what do you think about this?
```
ctx = {
    **page.form_spec.template_context(form),
    "application": application,
    "page": page,
}
```

Reply:
I'm using `|` later on, but otherwise, yes!